src/Public/js/nipsweb.librarypane.js

```diff
@@ -30,15 +30,22 @@ function updateCentralSearch()
           continue;
         }
         var classes = "";
+        var cleanStars = "";
+        var tooltip = "";
         if (!data[file].clean) {
           classes = classes + " unclean";
+          cleanStars = "**";
+          tooltip = "This track is explicit. Do not broadcast before 9PM."
+        }
+        if (!data[file].clean) {
+
         }
 
         $("#baps-channel-res").append(
           "<li id='" + data[file].album.recordid + "-" + data[file].trackid +
-          "' intro='" + data[file].intro + "'" +
+          "' intro='" + data[file].intro + "' title='" + tooltip + "'" +
```

Comment:
what happens here if `tooltip` isn't set? the title attribute is still set, does it still display something?